public/app/features/library-panels/components/LibraryPanelsView/actions.ts

```diff
@@ -16,9 +21,30 @@ interface SearchArgs {
   panelFilter?: string[];
   folderFilterUIDs?: string[];
   currentPanelId?: string;
+  isWidget?: boolean;
 }
 
 export function searchForLibraryPanels(args: SearchArgs): DispatchResult {
+  // Functions to support filtering out library panels per plugin type that have skipDataQuery set to true
+  const getAllPluginMetaList = () => getAllPanelPluginMeta();
```

Comment:
why do we rename it?

Reply:
I was trying to make the code "cleaner" by encapsulating the function and abstracting the implementation of the getAllPanelPlugin (). Now that I see it, it's overkill and might impact testing the method. I will just call the function instead 😄.